pkg/orchestration/wiring/kubeingress/plugin_test.go

```diff
@@ -111,17 +110,14 @@ func TestExtractKubeComputeDependency(t *testing.T) {
 
 	computeDep := wiringplugin.WiredDependency{
 		Name: deploymentName,
-		Type: apik8scompute.ResourceType,
 		Contract: wiringplugin.ResourceContract{
 			Shapes: []wiringplugin.Shape{
 				knownshapes.NewSetOfPodsSelectableByLabels(smith_v1.ResourceName(deploymentName), labels),
 			},
 		},
 	}
 
-	nonComputeDep := wiringplugin.WiredDependency{
-		Type: voyager.ResourceType("MiscellaneousResource"),
-	}
+	nonComputeDep := wiringplugin.WiredDependency{}
```

Comment:
Is this _correct_?

Reply:
Yes. It is used below to test the error scenario.